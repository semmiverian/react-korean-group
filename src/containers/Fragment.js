import React from 'react'

export default function Fragment() {
  return (
    <>
      <h2>hahahah</h2>
      <h3>hasjhdasjhdj</h3>
      <h3>kosasih</h3>
    </>
  )
}
