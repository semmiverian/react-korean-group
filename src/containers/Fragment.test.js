import React from 'react'
import { render } from '@testing-library/react'
import Fragment from './Fragment'

test('should work', () => {
  const { container } = render(<Fragment />)

  expect(container).toMatchSnapshot()
})