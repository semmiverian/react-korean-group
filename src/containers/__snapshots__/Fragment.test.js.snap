// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work 1`] = `
<div>
  <h2>
    hahahah
  </h2>
  <h3>
    hasjhdasjhdj
  </h3>
  <h3>
    kosasih
  </h3>
</div>
`;
